runner/runner.go

```diff
@@ -133,6 +133,9 @@ func (r *Runner) Run(ctx context.Context, args []string) int {
 		return 0
 	}
 
+	if r.releaseInfo.EnterpriseToken == "" {
```

Comment:
shouldn't we do this change it in main.go#31?
```
EnterpriseToken: config.GetString("ENTERPRISE_TOKEN", enterpriseToken),
```

Reply:
We can do it in main too.

But we need os.Getenv and not config.GetString (we don't want RSERVER_ prefix).

Reply:
don't worry about the RSERVER_ prefix, it looks up the env without it too ;)

Reply:
Oh kool. Pushing the change.

Reply:
Done